packages/xo-web/src/xo-app/new/network/index.js

```diff
@@ -61,6 +61,12 @@ const Item = ({ label, children, className }) => (
   </span>
 )
 
+const canSupportPrivateNetwork = (pool, pif) =>
+  (pif.isBondMaster || pif.physical || pif.vlan !== -1) &&
+  pif.mode !== 'None' &&
+  !pif.isBondSlave &&
+  pif.$host === (pool && pool.master)
```

Comment:
Can pool be `undefined`?

Reply:
No, it's a mandatory field to create a network.

Reply:
But if that's the case, the component should be properly unmounted and this function shouldn't be called any more. So it should actually never happen.

So if this function isn't supposed to be called with an `undefined` `pool`, I'd say don't protect it so that it doesn't hide other potential issues.

Reply:
Just tried it, you're right, you can't go to the network creation page when there's no pool connected.
If if it's already loaded, the page display "You do not have the permission"